packages/ui-svelte/src/Layout.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+  import './app.css'
+  import Nav from './Nav.svelte'
+  import Vercel from './Vercel.svelte'
+
+  export let title = 'Vercel Examples'
+  export let description = 'Lear how to use Vercel Middleware in Sveltekit'
```

Comment:
If this component library is designed to be used by multiple examples, then there probably shouldn't be a default value. Though if we do keep it, it's `SvelteKit` rather than `Sveltekit`

```suggestion
  export let description = 'Learn how to use Vercel Middleware in SvelteKit'
```